### ISSUE TITLE
Allow face deletion on picture

### ISSUE THREAD

**Issue by ced455** [photoview/photoview#269]
**Is your feature request related to a problem? Please describe.**
I have false postive face detection sometime.

**Describe the solution you'd like**
Be able to delete the face on the specific picture to clear up the people page 

**Describe alternatives you've considered**
Deleting the picture, but sometime I really want to keep it !

**Comment by viktorstrate** [comment]
I think the best solution might be to _hide_ false positives rather than delete them, and add a way to show hidden faces.
This would allow someone to recover/unhide faces easily, and also greatly simplifies the underlying logic.

I've also noticed that false positives tend to group together, and in that case the other false positives would automatically be added to a hidden face group, and you theirefore won't have to remove also remove them manually.

**Comment by ced455** [comment]
yep that's right , I have a face with mostly false negative so that should do it :)

**Comment by kkovaletp** [comment]
We need to think about the next side effect: if there is a hidden group and a new face is detected, there is a chance to merge it automatically with that group, so as a result, a person won't be visible, even while the face was detected. 

When we go to the implementation of the solution, we have to design the functionality to prevent or mitigate this case.
